webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -65,3 +68,21 @@ def inner(request, *args, **kwargs):
             raise
         return view(request, *args, **kwargs)
     return inner
+
+
+def track_api_pageview_decorator(view):
```

Comment:
You could make `track_api_pageview` work as both a function and a decorator but that's probably more exotic than it's worth.

I think it's good to have two things that do different things have two different names. Beyond that, I don't think this matters much in the grand scheme of things.